Formula/p/perl-dbd-mysql.rb

```diff
@@ -0,0 +1,95 @@
+# NOTE: This is an exception to Homebrew's cpan policy due to the extra work
+# needed to use macOS DBI and avoid overlinking to libraries like `zlib`.
+class PerlDbdMysql < Formula
+  desc "MySQL driver for the Perl5 Database Interface (DBI)"
+  homepage "https://dbi.perl.org/"
+  url "https://cpan.metacpan.org/authors/id/D/DV/DVEEDEN/DBD-mysql-5.010.tar.gz"
+  sha256 "2ca2ff39d93e89d4f7446e5f0faf03805e9167ee9b8a04ba7cb246e2cb46eee7"
+  license any_of: ["Artistic-1.0-Perl", "GPL-1.0-or-later"]
+  head "https://github.com/perl5-dbi/DBD-mysql.git", branch: "master"
+
+  depends_on "mysql" => :test
+  depends_on "mysql-client"
```

Comment:
Part of reason for PR is to also constrain `mysql-client` dependency to single formula. As of `DBD::mysql` v5, this module cannot use lighter-weight `mariadb-connector-c`.

Still considering whether to switch `mysql` test dependency to another option (e.g. `mariadb`) given Ventura will end up pulling in LLVM soon.